applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -159,6 +168,9 @@ void KratosStructuralMechanicsApplication::Register()
 
     // Register the membrane element
     KRATOS_REGISTER_ELEMENT( "MembraneElement3D3N", mMembraneElement3D3N )
+	KRATOS_REGISTER_ELEMENT( "MembraneElement3D4N", mMembraneElement3D4N )
+	KRATOS_REGISTER_ELEMENT( "Membrane1Element3D3N", mMembrane1Element3D3N )
+	KRATOS_REGISTER_ELEMENT( "Membrane1Element3D4N", mMembrane1Element3D4N )	
```

Comment:
Name not very descriptive (I mean the 1), maybe it is better to use a name based on the formulation employed, or something like LargeDisplacementMembrane